<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="12" hostname="BLU-LP-00004" name="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner" tests="13" failures="0" timestamp="2020-01-08T14:38:25 SGT" time="1.640" errors="1">
  <testcase name="tc02_invalidEmailValidPassword" time="0.000" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc02_invalidEmailValidPassword -->
  <testcase name="tc10_validMobileNumberInvalidPassword" time="0.000" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc10_validMobileNumberInvalidPassword -->
  <testcase name="tc11_invalidMobileNumberInvalidPassword" time="0.000" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc11_invalidMobileNumberInvalidPassword -->
  <testcase name="tc03_invalidMobileNumberValidPassword" time="0.000" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc03_invalidMobileNumberValidPassword -->
  <testcase name="tc04_withoutCredentials" time="0.000" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc04_withoutCredentials -->
  <testcase name="tc09_validEmailInvalidPassword" time="0.000" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc09_validEmailInvalidPassword -->
  <testcase name="preCondition" time="1.640" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <error type="org.openqa.selenium.WebDriverException" message="Connection refused: connect
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;BLU-LP-00004&#039;, ip: &#039;192.168.0.124&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_221&#039;
Driver info: driver.version: AndroidDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: Connection refused: connect
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'BLU-LP-00004', ip: '192.168.0.124', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_221'
Driver info: driver.version: AndroidDriver
at io.appium.java_client.remote.AppiumCommandExecutor.lambda$5(AppiumCommandExecutor.java:251)
at java.util.Optional.orElseGet(Unknown Source)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:250)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:41)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:336)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:37)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:88)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:98)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:94)
at bluAndroid.bluAndroid.util.BaseClass.getAppCapabilities(BaseClass.java:39)
at bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner.preCondition(BluLoginTestRunner.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:683)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:510)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.net.ConnectException: Connection refused: connect
at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
at java.net.PlainSocketImpl.connect(Unknown Source)
at java.net.SocksSocketImpl.connect(Unknown Source)
at java.net.Socket.connect(Unknown Source)
at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
at okhttp3.RealCall.execute(RealCall.java:77)
at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:105)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)
... 45 more
]]>
    </error>
  </testcase> <!-- preCondition -->
  <testcase name="tc06_notRegisteredMobileNumber" time="0.000" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc06_notRegisteredMobileNumber -->
  <testcase name="tc01_validMobileValidPassword" time="0.000" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc01_validMobileValidPassword -->
  <testcase name="tc07_notRegisteredEmail" time="0.000" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc07_notRegisteredEmail -->
  <testcase name="tc08_validEmailPassword" time="0.000" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc08_validEmailPassword -->
  <testcase name="splashScreen" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- splashScreen -->
  <testcase name="tc05_maximumInvalidLoginAttemptError" classname="bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner">
    <skipped/>
  </testcase> <!-- tc05_maximumInvalidLoginAttemptError -->
</testsuite> <!-- bluAndroid.bluAndroid.TestScripts.BluLoginTestRunner -->
